simulation/tests/it/xline.rs

```diff
@@ -0,0 +1,12 @@
+use curp_test_utils::init_logger;
+use simulation::xline_group::XlineGroup;
+use xline_client::types::kv::PutRequest;
+
+#[madsim::test]
+async fn basic_put() {
```

Comment:
Can we add more test cases for xline？

Reply:
This PR just add basic madsim test framework for xline, we can add more tests for this if needed

Reply:
Please add a todo comment here.